controller/array_action/messages.py

```diff
@@ -62,3 +62,8 @@
 ObjectIsStillInUseError_message = "Object {0} is still in use by {1}"
 
 InvalidCliResponseError_message = "Invalid CLI response. Details : {0}"
+
+NotEnoughSpaceInPoolError_message = "Not enough space in pool {0}"
+
+SizeOutOfRangeError_message = "requested size is out of limits. requested: {0}," \
+                              " limits are: min_in_byte: {1}, max_in_byte: {2}"
```

Comment:
consider removing the confusing `min_size` details until we add support for `limit_bytes`

Reply:
ok, done